src/System.Net.Http/src/System/Net/Http/Managed/ManagedHandler.cs

```diff
@@ -263,6 +263,11 @@ private HttpMessageHandler SetupHandlerChain()
         {
             HttpMessageHandler handler = new HttpConnectionHandler(_settings);
 
+            if (_settings._useCookies)
+            {
+                handler = new CookieHandler(CookieContainer, handler);
+            }
+
             if (_settings._useProxy &&
```

Comment:
Proxied requests bypass the base handler completely, no? So a proxied request would no longer get cookies? Then this needs to be Auth -> Cookies -> proxy -> handler.

Reply:
Reordered.  I'd thought you were saying otherwise from your offline comments, but I misunderstood.

Reply:
Sorry about that, my offline notes were more for identifying the problems than working out the full solutions.

Reply:
No problem :)